build.gradle

```diff
@@ -138,6 +138,15 @@ configure(subprojects) { p ->
 			maxGranularity 3
 		}
 
+		//show progress by displaying test classes, avoiding test suite timeouts
+		TestDescriptor last
+		afterTest { TestDescriptor td, TestResult tr ->
```

Comment:
I would suggest the following:
```groovy

        testLogging {
            displayGranularity 1
            showStackTraces = true
            exceptionFormat = 'full'
            events "STARTED", "PASSED", "FAILED", "SKIPPED"
        }
```

but okay :)

Reply:
I wanted to specifically avoid STARTED and PASSED, as it creates a _lot_ of noise (10K+ lines of logs that don't tell much). STARTED can be useful sometimes when dealing with a hanging test though, but in that case I would activate it temporarily rather than having it on all the time. Plus having the test class logged would already point us in some direction in that case.

Reply:
this was done as a compromise between adding `PASSED` to testEvents and logging nothing at all (which can fail the build as the tests now sometimes run for more than 10min total on CI)